index.html

```diff
@@ -0,0 +1,133 @@
+    notes or tempos.<br/>
+    The <a href="./app/features/lstm/">LSTM</a> folder contains 2 notebooks that allow us to test our LSTM model faster!
+    The file <a href="./app/lstm/LSTM_over_notes.ipynb">LSTM_over_notes.ipynb</a> and
+    the file <a href="./app/lstm/LSTM_on_binary.ipynb">LSTM_on_binary.ipynb</a> these two implementation of 2
+    model LSTM try for one to recognize the music thanks to
+    the notes of the file extracted from the MidiData classes. The second file is an attempt to implement an LSTM which
+    analyzes the file just by reading it without any analysis.
+    Regarding the .html and .py files that are in this folder they are just files generated by our notebook to be able
+    to
+    read us and share our code<br/>
+    The <a href="./app/features/cnn/">CNN</a> folder is organized in the same way as the LSTM folder it also contains an
+    exploration of another model of
+    Artificial Intelligence. We also find there the file
+    <a href="./app/cnn/CNN_over_notes.ipynb">CNN_over_notes.ipynb</a> which is a notebook to be able to easily
+    launch new Artificial Inteligence configuration it contains a model that tries to analyze the notes of a music!
+    Regarding the organization of the rest of the folder we decided to push the <a href="data/maestro-v2.0.0">dataset</a>  with the files that allow us to
+    read it with <a href="./data/maestro-v2.0.0/maestro-v2.0.0.csv">maestro-v2.0.0.json</a>.
+</p>
+<h2>Responsibility for the Code</h2>
+<ul>
+    <li>
+        <b>Antoine Lefebvre:</b><br/>
+        For this project, he was in charge of extracting features from the music cheat. He uses
+        different libraries to parse MIDI files in order to deduce information that can be interesting for learning and
+        therefore author recognition.
+    </li>
+    <li>
+        <b>Armand Mégrot:</b><br/>
+        He created an LSTM Neural Network to be able to recognize the music through Antoine's features.
+        Armand also configured TensorflowGPU to be able to train Artificial Intelligence faster.
+        Some models would have taken several days to train if we couldn't use Armand's graphics card.
+    </li>
+    <li>
+        <b>Thomas Burgaud:</b><br/>
+        As explained above Thomas is busy exploring a music classification system through a Convolutional Neural
+        Network.
+        So he used Antoine's work to try to learn the interesting features to deduce the author of a music through these
+        notes.
+    </li>
+</ul>
+<h2>Result</h2>
+<p>The goal of this project was to explore the classification of music through several models of Artificial
+    Intelligence. Some of the results are not very convincing since their use is not useful in this kind of problem. The
+    results are very different between Artificial Intelligence since their implementation is very different depending on
+    their structure, their strengths and weaknesses.</p>
```

Comment:
```suggestion
    Intelligence. Some of the results are not very convincing the models are not tailored for this specific kind of problem. The
    results are very different between Artificial Intelligences since their implementations, structures, strengths and weaknesses are not the same.</p>
```